### PR TITLE
AzureStack - Cleaning up the old 2015-06-15 swagger for the support of CLI/Python sdk

### PR DESCRIPTION
**Pull request by bganapa**
Changing descriptions, operation status response, changing CreateOption and CachingType as a seperate model

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
I'm reviewing this today.

File: arm-compute/2015-06-15/swagger/compute.json [hunk 1]
```diff
@@ -2451,26 +2590,27 @@
...
         "virtualMachines": {
           "type": "array",
           "items": {
             "$ref": "#/definitions/SubResource"
           },
-          "description": "Gets or sets a list containing reference to all Virtual Machines  created under this Availability Set."
+          "description": "A list of references to all virtual machines in the availability set."
         },
         "statuses": {
+          "readOnly": true,
```
Comment (jhendrixMSFT) [inline arm-compute/2015-06-15/swagger/compute.json]: Is the service changing the way this field is handled or is this a bug in the swagger that's being corrected?

File: arm-compute/2015-06-15/swagger/compute.json [hunk 2]
```diff
@@ -1006,11 +1028,17 @@
           }
         ],
         "responses": {
+          "200": {
```
Comment (jhendrixMSFT) [inline arm-compute/2015-06-15/swagger/compute.json]: Is the service being changed to now return 200 in cases where it previously did not?

**Comment by jhendrixMSFT** [comment]
@bganapa ping :)

**Comment by bganapa** [comment]
@jhendrixMSFT I will work on the comments on Thursday, sorry for the delay

File: arm-compute/2015-06-15/swagger/compute.json (hunk 2 above)
Comment (bganapa) [inline arm-compute/2015-06-15/swagger/compute.json]: @jhendrixMSFT, Yes here and in other places, we are just correcting the swagger.  The service has been returning the 200, the swagger was not correct for the older specs

File: arm-compute/2015-06-15/swagger/compute.json (hunk 1 above)
Comment (bganapa) [inline arm-compute/2015-06-15/swagger/compute.json]: @jhendrixMSFT , here as well we are just correcting the swagger, the service has not changed

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**